application/views/backend/settings.php

```diff
@@ -108,6 +108,7 @@
                                 </label>
                                 <select class="form-control" id="date-format" data-field="date_format">
                                     <option value="DMY">DMY</option>
+                                    <option value="DDMY">DDMY</option>
                                     <option value="MDY">MDY</option>
                                     <option value="YMD">YMD</option>
                                 </select>
```

Comment:
I'd suggest you change it to this:

```
<option value="DMY">D/M/Y</option>
<option value="DDMY">D.M.Y</option>
<option value="MDY">M/D/Y</option>
<option value="YMD">Y/M/D</option>
```
This would reflect the chosen date format better.